File: solutions/cpp/etl/1/etl.cpp
#include "etl.h"

namespace etl {

std::map<char, int> transform(const std::map<int, std::vector<char>>& points_table) {
    std::map<char, int> res {};
    for (const auto& [key, letters] : points_table) {
        for (char l : letters) {
            res[std::tolower(l)] = key;
        }
    }
    return res;
}


}  // namespace etl

File: solutions/csharp/etl/1/Etl.cs
public static class Etl
{
    
    
    public static Dictionary<string, int> Transform(Dictionary<int, string[]> old)
    {
        Dictionary<string, int> n = new();
        foreach(var kv in old){
            int points = kv.Key;
            foreach(var value in kv.Value){
                n.Add(value.ToLower(), points);
            }
        }
        return n;
    }
}